### PR TITLE
Update FOXE-FOXEERF405V2.config

### PR DESCRIPTION
**Pull request by sugaarK**
fix missing timer and pino

@Johnsnow-foxeer  want to check

### CODE REVIEW

File: configs/default/FOXE-FOXEERF405V2.config [hunk 1]
```diff
@@ -116,6 +120,9 @@ feature RX_SERIAL
 feature TELEMETRY
 feature OSD
 
+# aux
+aux 2 40 2 1600 2100 1
```
Comment (haslinghuis) [inline configs/default/FOXE-FOXEERF405V2.config:124]: Suggest adding `set box_user_1_name` to clarify what this setting is used for.

**Review by haslinghuis — CHANGES_REQUESTED** [review]

File: configs/default/FOXE-FOXEERF405V2.config (hunk 1 above)
Comment (sugaarK) [reply configs/default/FOXE-FOXEERF405V2.config:124]: Can you commit it @haslinghuis  I’m not with my computer all day

Comment (haslinghuis) [reply configs/default/FOXE-FOXEERF405V2.config:124]: What function is it used for? VTX_POWER ?

Comment (sugaarK) [reply configs/default/FOXE-FOXEERF405V2.config:124]: Pino switch

Comment (haslinghuis) [reply configs/default/FOXE-FOXEERF405V2.config:124]: For what function? So we can label it

Comment (sugaarK) [reply configs/default/FOXE-FOXEERF405V2.config:124]: Yeah I guess VTX power
Never use it 😂

Comment (Johnsnow-foxeer) [reply configs/default/FOXE-FOXEERF405V2.config:124]: set box_user_1_name = CAM_SWITCH, it is cam switcher, we integrate the 2 way video switcher in the FC

**Review by sugaarK — COMMENTED** [review]
Thanks @haslinghuis

**Review by haslinghuis — APPROVED** [review]

**Review by nerdCopter — APPROVED** [review]

**Pull request merged**